crates/rome_js_formatter/src/js/expressions/call_arguments.rs

```diff
@@ -35,25 +34,68 @@ impl FormatNodeRule<JsCallArguments> for FormatJsCallArguments {
             );
         }
 
-        // particular formatting for hooks
-        if let Some((first_argument, second_argument)) = is_react_hook_with_deps_array(&args)? {
-            write!(
-                f,
-                [
-                    l_paren_token.format(),
-                    first_argument.node().format(),
-                    first_argument.trailing_separator().format(),
-                    space_token(),
-                    second_argument.node().format(),
-                ]
-            )?;
-
-            // we don't want to print the trailing separator, so if it's present, we remove it
-            if let Some(separator) = second_argument.trailing_separator()? {
-                write!(f, [format_removed(separator)])?;
+        let mut iter = args.iter();
+        let first_argument = iter.next();
+        let second_argument = iter.next();
+        let third_argument = iter.next();
+
+        if let (Some(first_argument), Some(second_argument)) = (first_argument, second_argument) {
+            let first_argument = first_argument?;
+            let second_argument = second_argument?;
+
+            if let Some(call_expression) = node
+                .syntax()
+                .parent()
+                .as_ref()
+                .and_then(|node| JsCallExpression::cast(node.clone()))
+            {
+                let callee = call_expression.callee()?;
+
+                if is_framework_test_call(IsTestFrameworkCallPayload {
+                    first_argument: &first_argument,
+                    second_argument: &second_argument,
+                    third_argument: &third_argument,
+                    arguments_len,
+                    callee: &callee,
+                })? {
+                    return write!(
+                        f,
+                        [
+                            l_paren_token.format(),
+                            format_with(|f| {
+                                let separated =
+                                    args.format_separated(JsSyntaxKind::COMMA).with_options(
+                                        FormatSeparatedOptions::default()
+                                            .with_trailing_separator(TrailingSeparator::Omit),
+                                    );
```

Comment:
Nit: I simplified this API a bit. You can now do
```suggestion
                                    args.format_separated(JsSyntaxKind::COMMA).with_trailing_separator(TrailingSeparator::Omit);
```